docs/guides/templates.md

```diff
@@ -24,6 +24,34 @@ npx create-remix --template remix-run/remix/templates/remix-javascript
 
 This is a great place to start if you're just looking to try out Remix for the first time. You can always extend this starting point yourself or migrate to a more advanced template later.
 
+### Official Templates
```

Comment:
I lifted this section from below, it felt weird that our hierarchy was flipping between templates and stacks:

```
Templates
  Basic Templates
  Stacks
    Official Stacks
    Community Stacks
  Regular Templates and Examples
    Third Party Templates
```

The new hierarchy is:

```
Templates and Stacks
  Templates
    Official Templates
    Third-Party Templates
    Examples
  Stacks
    Official Stacks
    Community Stacks
```